docs/chain/collator.md

```diff
@@ -574,21 +547,22 @@ docker run -p 127.0.0.1:9933:9933 -v ~/data:/data \
 </TabItem>
 </Tabs>
 
-## Known Issues
+:::info
 
-The following are known issues and the possible solutions:
+The same process that includes generating and registering sessions keys and joining the collator candidate pool must be re-executed for Spiritnet.
 
-- running "cargo update -p environmental" could fix something
-- Using the commands create a `/Data/` directory the folder could possibly be owned by the root user. Switching the ownership of the folder with `chown [options] user:group filename`. Solution found [here](https://serverfault.com/questions/106713/how-to-change-ownership-of-a-folder-using-terminal)
+:::
 
 ## Troubleshooting
 
-There are a few things that you can check to make sure everything is setup correctly.
+There are a few things that can checked to make sure everything is setup correctly.
```

Comment:
i would keep this note:

```
Check that your node is fully synced with the relaychain & parachain. (best and finalised block number is equal to the one shown in the polkadot apps & on subscan)
```

Reply:
It is, just a few points below, precisely point 3. I just switched the points around as to first suggest to check if everything is good, and if not, rule out potential causes before trying to upgrade the hardware. I think it is a more logical flow to follow.

Reply:
ok!